client/upe-onboarding-wizard/upe-preview-methods-selector/setup-complete-task.js

```diff
@@ -22,19 +20,27 @@ const SetupCompleteMessaging = () => {
 		return null;
 	}
 
-	const addedPaymentMethodsCount =
-		enabledMethodsCount - initialMethods.length;
+	// we need to check that the type of `enableUpePreviewPayload` is an object - it can also just be `true` or `undefined`
+	let addedPaymentMethodsCount = 0;
+	if (
+		typeof enableUpePreviewPayload === 'object' &&
+		enableUpePreviewPayload.initialMethods
```

Comment:
```suggestion
	if (
		enableUpePreviewPayload?.initialMethods
	)
```
Does this work? Also, I am curious to know what are the cases that make `enableUpePreviewPayload` either `true` or `undefined`.

Reply:
Oh, thanks! I wasn't sure that the project was setup with optional chaining 👍 Adjusted in ba66bfc1